demo/demodesk/config/settings.py

```diff
@@ -39,9 +33,10 @@
 # an internal demo you don't need such security, but please
 # remember when setting up your own development / production server!
 
+# Default teams mode to enabled unless overridden by an environment variable set to "false"
+HELPDESK_TEAMS_MODE_ENABLED=False if os.getenv("HELPDESK_TEAMS_MODE_ENABLED", "true").lower() == "false" else True
```

Comment:
I think we can explicitly choose to use Teams in the demodesk in order to show that feature
```suggestion
HELPDESK_TEAMS_MODE_ENABLED=True
```

Reply:
Sorry, I read the comment above after. We can keep that but simplify it like this:
```suggestion
HELPDESK_TEAMS_MODE_ENABLED=os.getenv("HELPDESK_TEAMS_MODE_ENABLED", "true").lower() == "true"
```